tests/drivers/build_all/sensor/i2c.dtsi

```diff
@@ -559,6 +559,15 @@ test_i2c_vcnl4040: vcnl4040@51 {
 	int-gpios = <&test_gpio 0 0>;
 };
 
+apds_9306: apds9306@52 {
+	compatible = "avago,apds9306";
+	reg = <0x52>;
+	status = "okay";
+	gain = <1>;
+	resolution = <13>;
+	frequency = <2000>;
+};
```

Comment:
Move to the bottom of the file and increment the address

Reply:
So I should pick the next free address, regardless if this is the real address or not?